src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -187,19 +187,6 @@ public void moveTokenStatsToS3() throws IOException {
         log.info("Finished the cronjob to move token stats to s3.");
     }
 
-    private void calculateUsageSummary(UsageSummary usageSummary, String key, int count, String time) {
-        // Deal with year summary
-        usageSummary.getYear().put(key, usageSummary.getYear().getOrDefault(key, Long.valueOf(0)) + count);
-        // Deal with month summary
-        if (!usageSummary.getMonth().containsKey(time)) {
-            usageSummary.getMonth().put(time, new JSONObject());
-        }
-        if (!usageSummary.getMonth().get(time).containsKey(key)) {
-            usageSummary.getMonth().get(time).put(key, new Integer(0));
-        }
-        usageSummary.getMonth().get(time).put(key, (Integer) usageSummary.getMonth().get(time).get(key) + (Integer) count);
-    }
-
```

Comment:
My changes caused a type error and this code isn't used so I chose to delete it.